pokemongo_bot/__init__.py

```diff
@@ -100,7 +100,8 @@ def update_web_location(self, cells=[], lat=None, lng=None, alt=None):
                                                      gym_latitude=fort.get('latitude'),
                                                      gym_longitude=fort.get('longitude'))
                             response_gym_details = self.api.call()
-                            fort['gym_details'] = response_gym_details['responses']['GET_GYM_DETAILS']
+                            if response_gym_details['responses'] is not None:
+                                fort['gym_details'] = response_gym_details['responses']['GET_GYM_DETAILS']
```

Comment:
@douglascamata: Ah, thanks! Unfortunately, having never looked at Python source before 20 minutes ago, I can only guess that your comment shows a way to set a default value of an empty result set? Is that right?